CommonLibSF/include/SFSE/Utilities.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "REL/Relocation.h"
+#include "SFSE/API.h"
+
+namespace SFSE
+{
+	namespace stl
+	{
+		template <class T, std::size_t Size = 5>
+		constexpr void write_thunk_call() noexcept
+		{
+			SFSE::AllocTrampoline(14);
+			auto& trampoline = SFSE::GetTrampoline();
+			T::func = trampoline.write_call<Size>(T::address, T::Thunk);
+		}
+
+		template <class TDest, class TSource>
```

Comment:
Aren't using `To` and `From` better? CLib has never used `TDest` and `TSource` before, but `To` and `From` has been used a lot.

Reply:
[#108dc2d](https://github.com/Starfield-Reverse-Engineering/CommonLibSF/pull/44/commits/108dc2d1e527d6c705fa1b3de456253060f393d2)